networkx/generators/tests/test_directed.py

```diff
@@ -56,6 +56,33 @@ def test_create_using_keyword_arguments(self):
         pytest.raises(ValueError, scale_free_graph, 100, alpha=-0.3)
         pytest.raises(ValueError, scale_free_graph, 100, beta=-0.3)
         pytest.raises(ValueError, scale_free_graph, 100, gamma=-0.3)
+        G = nx.DiGraph()
+        G.add_node(0)
+
+        def kernel(x):
+            return x
+
+        assert nx.is_isomorphic(gn_graph(1), G)
+        assert nx.is_isomorphic(gn_graph(1, kernel=kernel), G)
+        assert nx.is_isomorphic(gnc_graph(1), G)
+        assert nx.is_isomorphic(gnr_graph(1, 0.5), G)
```

Comment:
An organizational nit, but this should be broken out into it's own test since it doesn't have anything to do with the `create_using` kwarg.